.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+-   repo: https://github.com/Yelp/detect-secrets
+    rev: v1.2.0
+    hooks:
+    -   id: detect-secrets
+        args: []
+        exclude: .*/tests/.*
```

Comment:
someone may still commit a test file containing a secret right?

Reply:
removed the exclude